EIPS/eip-6036.md

```diff
@@ -0,0 +1,116 @@
+---
+eip: 6036
+title: Subscribeable NFT Extension
+description: 721sub is a subscribable nft protocol that allows users to enjoy their benefits for a certain period of time.
+author: Diven (@aboutmydreams), Kaso (@kasoqian), Jason (@350306878)
+discussions-to: https://github.com/ethereum/EIPs/pull/6036
+status: Draft
+type: Informational
+created: 2022-08-17
+requires: 165, 721
+---
+
+## Abstract
+
+Subscribeable NFT Extension is a standard that allows more platforms to directly use the interface to determine whether a user is still within the enjoyment time of a certain benefit
```

Comment:
This is unclear. I think you meant:
```suggestion
This EIP standardizes an interface extending [EIP-721](./eip-721.md) that allows for NFTs to be temporarily purchased.
```
This still needs to be a bit longer, though.